networking/topology.md

```diff
@@ -31,7 +31,7 @@ In the future, this VNet might hold more subnets like [ACI Provider instance] su
 
 | Subnet                                                 | Upgrade Node | Nodes/VMs/Instance | % Seasonal scale out | +Nodes/VMs | Max Ips/Pods per VM/Node | [% Max Surge] | [% Max Unavailable] | +Ips/Pods per VM/Node | Tot. Ips/Pods per VM/Node | [Azure Subnet not assignable Ips factor] | [Private Endpoints] | [Minimum Subnet size] | Scaled Subnet size | [Subnet Mask bits] | Cidr           | Host        | Broadcast     |
 |--------------------------------------------------------|--------------|--------------------|------------------|------------|--------------------------|---------------|---------------------|-----------------------|---------------------------|------------------------------------------|---------------------|-----------------------|--------------------|--------------------|----------------|-------------|---------------|
-| AKS System and User Nodepool Subnet                    | 2            | 5                  | 200              | 10         | [30]                     | 100           | 0                   | 30                    | 60                        | 5                                        | 2                   | 374                   | 984                | 22                 | 10.240.0.0/22  | 10.240.0.0  | 10.240.3.255  |
+| AKS System and User Nodepool Subnet                    | 2            | 6                  | 200              | 12         | [30]                     | 100           | 0                   | 30                    | 60                        | 5                                        | 2                   | 374                   | 984                | 22                 | 10.240.0.0/22  | 10.240.0.0  | 10.240.3.255  |
```

Comment:
Thank you for your pull request. The number of VMSS instances that are deployed by default is five as this line indicates. Three for the system node pool, and two for the user node pool.  Where is six/ten coming from?

Reply:
bump @hartou ^

Reply:
Bumping again for clarity @hartou ^